README.md

```diff
@@ -174,3 +174,28 @@ This will then generate a major release and add a `BREAKING CHANGES` section in
 
 # Bug reports
 Bug reports must be submitted through Zendesk's standard support channels: https://www.zendesk.com/contact/
+
+# Quick Setup Recall (Running and testing theme locally)
+
+NOTE: This guide is to quickly setup and run the theme locally. To know more about how to customize the theme, See all the details above.
+
+1. Create a trial account on https://www.zendesk.com/. This will let you create a subdomain Zendesk account that you can manage.
+2. Make sure to opt-in for Guide integration once your trial account setup is done.
+3. Create API keys for running the server locally and previewing them on your subdomain. `<YOUR_ZENDESK_DOMAIN>/admin/apps-integrations/apis/zendesk-api/settings`.
+4. Enable Help Center for your account through `<YOUR_ZENDESK_DOMAIN>/hc/admin/general_settings`
+5. Install [ZAT](https://developer.zendesk.com/documentation/apps/zendesk-app-tools-zat/installing-and-using-zat/)
```

Comment:
We need to mention ZAT CLI too. like we did [here](https://github.com/mitodl/mitxonline-zendesk-theme-updated/commit/0f4f9eca92df7203f7a9365ceb17d153d75cbb0d#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R124).

Reply:
Sorry, I picked these changes from the [Bootcamps Zendesk](https://github.com/mitodl/bootcamps-zendesk-theme), I will open a PR for the Bootcamps to fix it there as well.

Reply:
Fixed in https://github.com/mitodl/micromasters-zendesk-theme/pull/1/commits/c588df8a6399e48407c0f8014ed99f9ec382ee4b